packages/core/test/interpreter.test.ts

```diff
@@ -2019,9 +2021,9 @@ Event: {\\"type\\":\\"SOME_EVENT\\"}"
                 target: 'gone',
                 actions: [
                   // TODO: type these correctly in TContext
-                  stop((ctx) => (ctx as any).machineRef),
-                  stop((ctx) => (ctx as any).promiseRef),
-                  stop((ctx) => (ctx as any).observableRef)
+                  stop((ctx: any) => ctx.machineRef),
+                  stop((ctx: any) => ctx.promiseRef),
+                  stop((ctx: any) => ctx.observableRef)
```

Comment:
Is it possible to infer these?

Reply:
You ask, I deliver: https://github.com/statelyai/xstate/pull/3802/commits/683e616712ecf53549b50c7c13777c48c3b25459